src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -480,7 +480,7 @@ public void Deconstruct(out int a, out string b, out params int[] c)
 
             var comp = CreateStandardCompilation(source);
             comp.VerifyDiagnostics(
-                // (4,58): error CS8404:  The parameter modifier 'params' cannot be used with 'out' 
+                // (4,58): error CS8404:  The parameter modifier 'params' cannot be used after the modifier 'out' 
```

Comment:
> I think this is irrelevant. The message doesn't mean the opposite is true.
This changed because parameters now have to appear in certain orderings.

To me, the error implies that I can change the order of modifiers and everything will be fine. We are adding order dependency only for certain modifiers, I believe we should add a new error that will be used only for these situations, we should use old error for cases when modifiers cannot be combined regardless of the order.